lib/formatters/stringFormatter.js

```diff
@@ -200,6 +203,8 @@ function formatter(messages, source, cwd) {
 		return row;
 	});
 
+	const hasNonAsciiChar = messages.some((msg) => NON_ASCII_OR_WS_PATTERN.test(msg.text));
```

Comment:
[suggset] We can now use a [unicode character class escape](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape) because the spec has been available since Node.js 10.

So, could you simplify the regex like this?

```suggestion
	const hasNonAsciiChar = messages.some((msg) => /\P{ASCII}/u.test(msg.text));
```